<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
<widget class="QWidget" name="Chat">
    <property name="geometry">
        <rect>
            <x>0</x>
            <y>0</y>
            <width>400</width>
            <height>300</height>
        </rect>
    </property>
    <property name="windowTitle">
        <string>Form</string>
    </property>

    <layout class="QVBoxLayout" name="verticalLayout_1">

        <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="3,1">
                <item>
                    <widget class="QLabel" name="modelLabel">
                        <property name="text">
                            <string></string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="DisconnectButton">
                        <property name="text">
                            <string>Disconnect</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </item>


        <item>

            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">

                <item>
                    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,3">
                        <item>
                            <widget class="QGroupBox" name="ConversationsGroup">
                                <!-- <property name="title">
                                    <string>Conversations</string>
                                </property> -->
                                <layout class="QVBoxLayout" name="verticalLayout_278" stretch="1,6">
                                    <item>
                                        <widget class="QPushButton" name="NewConversationButton">
                                            <property name="text">
                                                <string>New</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QListWidget" name="ConversationsListWidget"/>
                                    </item>
                                </layout>
                            </widget>
                        </item>

                    </layout>
                </item>

                <item>
                    <widget class="QGroupBox" name="ActiveChatGroup">

                        <layout class="QVBoxLayout" name="verticalLayout" stretch="11,2,1">
                            <item>
                                <widget class="QTextEdit" name="MessageDisplay"/>
                            </item>
                            <!-- <item>
                                <widget class="QListWidget" name="MessageDisplayListWidget"/>
                            </item> -->
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
                                    <item>
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>[Ctrl + S] to send</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="SendPromptButton">
                                            <property name="text">
                                                <string>Send Prompt</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QPlainTextEdit" name="PromptEditor"/>
                            </item>
                        </layout>
                    </widget>
                </item>
            </layout>
        </item>
        
    </layout>

</widget>
<resources/>
<connections/>
</ui>
